public/locations-list-map.js

```diff
@@ -1090,11 +1090,11 @@ function updateStats($elem, count, states) {
  *
  * Result:  "1.000,2"
  */
-function number_format(number, decimal_places, dec_seperator, thou_seperator) {
+function number_format(number, decimal_places, dec_separator, thou_separator) {
```

Comment:
is it just me or is there no diff here? Any idea why Github is being so weird?

Reply:
Github is fine.  I corrected the spelling

Reply:
ohhhh. Good catch! Somehow I missed that looking at it.